src/components/App/CodeSplit/CodeSplit.jsx

```diff
@@ -7,28 +8,36 @@ import { FadeTransform } from 'react-animation-components';
 const loadMyDep = () =>
     import(/* webpackChunkName: "myDependency" */ './myDep.js');
 
-const CodeSplit = () => (
-    <Bundle load={loadMyDep}>
-        {myDep => (
-            <FadeTransform in timeout={0}>
-                <Text theme="headline" tag="h1">
-                    Code Splitting
-                </Text>
-                <Text tag="p" onClick={myDep}>
-                    I fade in once my dependency is loaded.
-                </Text>
-                <Text theme="subheadline">
-                    👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍
-                    👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍
-                    👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍
-                    👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍
-                    👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍
-                    👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍 👍
-                    👍 👍 👍 👍 👍 👍 👍 👍 👍 ❤️
-                </Text>
-            </FadeTransform>
-        )}
-    </Bundle>
-);
+function CodeSplit() {
+    return (
+        <Bundle load={loadMyDep}>
+            {myDep => (
+                <Fragment>
+                    <Helmet>
```

Comment:
I would move Helmet out of the `Bundle` render prop since it doesn't call this until the dependency is loaded. No need to wait to update the `head` until then